backend/ecommerce/schema.py

```diff
@@ -143,6 +148,41 @@ def mutate(self, info, name, email, phone, password):
         )
 
 
+class LikeReview(graphene.Mutation):
+    id = graphene.String()
+
+    class Arguments:
+        reviewId = graphene.String()
+
+    @login_required
+    def mutate(self, info, reviewId):
+        user = info.context.user
+        review = Review.objects.get(pk=reviewId)
+        like = Like(user=user, review=review)
+        like.save()
+
+        return LikeReview(id=like.id)
+
+
+class UnlikeReview(graphene.Mutation):
+    id = graphene.String()
+
+    class Arguments:
+        reviewId = graphene.String()
```

Comment:
We will be having likeId on front-end. Use that only

Reply:
No, we won't have likeId on frontend:
The user will click on the :+1: button again to unlike, so we know on which review he pressed and not the exact like (we can filter all likes but why do extra work)

Reply:
yeah got you.